src/NoSignInIAuthClient.ts

```diff
@@ -47,12 +47,12 @@ export class NoSignInIAuthClient implements FrontendAuthorizationClient {
 
     const response = await fetch(userURL);
     const body = await response.json();
-    const tokenJson = {
+    const tokenJson: AccessTokenProps = {
       ...await body,
       // eslint-disable-next-line @typescript-eslint/naming-convention
-      _userInfo: { id: "MockId" },
```

Comment:
Should getDevAccessToken() also have the same change?
In this case we're setting the authClient for iModelApp which seems to be bound to no underscores now. But for developer.bentley.com, this doesn't seem to be the case.

Reply:
It seems like APIs from developer.bentley.com still assumes the old tokens with underscores, this should be fine for now.

Reply:
For uniformity, let's change getDevAccessToken() for this change as well.

Reply:
Makes sense

Reply:
Was also able to remove the lint `eslint-disable-next-line`